scipy/stats/stats.py

```diff
@@ -4363,10 +4363,8 @@ def ks_2samp(data1, data2):
     data2 = np.sort(data2)
     n1 = data1.shape[0]
     n2 = data2.shape[0]
-    data_all = np.concatenate([data1, data2])
-    cdf1 = np.searchsorted(data1, data_all, side='right') / (1.0*n1)
-    cdf2 = np.searchsorted(data2, data_all, side='right') / (1.0*n2)
-    d = np.max(np.absolute(cdf1 - cdf2))
+    from . import _stats
```

Comment:
I'll move the import up.
Regarding the filename, I was hoping that `_stats.pyx` would ultimately host Cython accelerations for all stats related functions that may benefit from it.